lib/content-services/src/lib/interfaces/library-membership.interface.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface LibraryMembershipToggleEvent {
+    updatedEntry?: any;
+    shouldReload: boolean;
+    i18nKey: string;
+}
+
+export interface LibraryMembershipErrorEvent {
```

Comment:
why don't we have 2 files ?

Reply:
Because the both interfaces are related to the Library Membership, should we create one single files for every interfaces?